Snippets/Core/Core_8/OperationProperties/OperationPropertiesUsage.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace Core8.OperationProperties
```

Comment:
this snippet will fail compilation until https://github.com/Particular/NServiceBus/pull/6349 is merged and a new version of core is pushed to myget.